Python/Product/Cookiecutter/Attributes.cs

```diff
@@ -26,7 +27,7 @@ internal sealed class SRDisplayNameAttribute : DisplayNameAttribute {
             _name = name;
         }
 
-        public override string DisplayName => Strings.ResourceManager.GetString(_name);
+        public override string DisplayName => Strings.ResourceManager.GetString(_name, CultureInfo.InvariantCulture);
```

Comment:
We don't want to pass `InvariantCulture` here.

When using the generated code in Strings.Designer.cs, you can see that all calls are like this one:
```
ResourceManager.GetString("SearchPage_LinksGitHubIssues", resourceCulture);
```

`resourceCulture` is private but there's an accessor `Culture`.

So the code should be:
```
Strings.ResourceManager.GetString(_name, Strings.Culture)
```

Reply:
Same for all the other `ResourceManager.GetString` calls.

Reply:
Will be modified in next commit